README .md

```diff
@@ -0,0 +1,53 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
```

Comment:
- [OPTIONAL] Great job documenting your project! I would suggest removing the space before the `.md` extension in the file name `README .md` in order to display it properly.